local-test-configuration/scripts/initialize-chain.ts

```diff
@@ -2,17 +2,17 @@ import { readFileSync } from 'node:fs';
 import { join } from 'node:path';
 
 import { encode } from '@api3/airnode-abi';
+import {
+  AirseekerRegistry__factory as AirseekerRegistryFactory,
+  AccessControlRegistry__factory as AccessControlRegistryFactory,
+  Api3ServerV1__factory as Api3ServerV1Factory,
+} from 'api3-contracts';
```

Comment:
I suspected name of the package and asked @bbenligiray , and he said we shouldn't use it. I think the package we should use is [@api3/contracts@2.0.0-beta13](https://www.npmjs.com/package/@api3/contracts/v/2.0.0-beta13).

Reply:
Thanks.